config/auth/config.example.yaml

```diff
@@ -4,7 +4,7 @@ app:
   secret: <secret>
 
 database:
-  host: localhost
+  host: local-db
```

Comment:
why we change to local-db?

Reply:
```docker-compose.yaml``` sets the backend, auth, file in the same network ```johnjud-local```. So, ```localhost``` cannot be used, and container name is used instead

Reply:
But in johnjud-auth repo, for example, there are no networks, so the config in that repo would still be ```localhost```

Reply:
ok, I see your point. not sure that if it work the same since we already exposed port to host port. btw, I fine in both way.